docs/api/rest_api/rest_api_reference/input/ez-orders.raml

```diff
@@ -75,6 +75,35 @@ displayName: Orders
                     description: Error - the user has no permission to read orders.
                 404:
                     description: Error - the order does not exist.
+                    
+        /shipments:
+            get:
+                displayName: Load shipments
```

Comment:
It looks wrong. According to https://github.com/ibexa/shipping/pull/40/files#diff-b50dddb91d44700cc0b64f88a159499eb019b780afc5ec10913f8ed0cd145eef this root is used with POST to create a shipment.